examples/llama-llm-chatbots/llama-13b-upload.yaml

```diff
@@ -0,0 +1,31 @@
+resources:
+  accelerators: A100:2
+  disk_size: 1024
+
+file_mounts:  # /remote/path: /local/path
+  ~/sky_workdir/ckpt/tokenizer.model: /tmp/llama/tokenizer.model
+  ~/sky_workdir/ckpt/13B: /tmp/llama/13B
```

Comment:
Correct me if I missed something, but thought we decided to use bucket upload, instead of rsync?